packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -573,26 +573,35 @@ export default defineComponent({
     getUploadItemMessage(item) {
       const error = this.errors[item.meta.uploadId]
 
-      if (error) {
-        // TODO: Remove code as soon as https://github.com/tus/tus-js-client/issues/448 is solved
-        let errorMessage = this.$gettext('Unknown error')
-        if (error.message.includes('response code: 507')) {
-          errorMessage = this.$gettext('Quota exceeded')
-        }
-        if (error.message.includes('precondition failed:')) {
-          errorMessage = this.$gettext('Parent folder does not exist')
+      if (!error) {
+        return
+      }
+
+      //TODO: Remove extraction code as soon as https://github.com/tus/tus-js-client/issues/448 is solved
+      const formatErrorMessageToObject = (errorMessage) => {
+        let responseCode = errorMessage.match(/response code: (\d+)/)?.[1]
```

Comment:
any reason why `let` and not `const`

Reply:
sure, will be parsed to int (some lines under it), and parsing something else like null or undefined will result in NaN